pkg/cmd/grafana/apiserver/server.go

```diff
@@ -170,9 +170,6 @@ func (o *APIServerOptions) Config() (*genericapiserver.RecommendedConfig, error)
 		openapi.GetOpenAPIDefinitionsWithoutDisabledFeatures(defsGetter),
 		openapinamer.NewDefinitionNamer(Scheme))
 
-	// Add the custom routes to service discovery
-	serverConfig.OpenAPIV3Config.PostProcessSpec3 = grafanaAPIServer.GetOpenAPIPostProcessor(o.builders)
```

Comment:
@ryantxu any thoughts on how to replace this now that v0.29.0 has been released?

Reply:
https://github.com/kubernetes/kube-openapi/pull/439
and
https://github.com/ryantxu/kube-openapi/tree/v29-with-hook